cvat-canvas/README.md

```diff
@@ -104,7 +108,9 @@ Canvas itself handles:
         dragCanvas(enable: boolean): void;
         zoomCanvas(enable: boolean): void;
 
+        mode(): void;
```

Comment:
I see that `mode()` method returns Mode object

https://github.com/opencv/cvat/blob/feebec2670186f8f56f4314cb45d596d5dc13e8a/cvat-canvas/src/typescript/canvas.ts#L137-L139

Why did it change in documentation and Canvas interface?

Reply:
It seems I did it accidently